### PR TITLE
add strategies for pendle pools

### PR DESCRIPTION
**Pull request by 0xKMG**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/airpuff exports TVL: 
        
 
 
    arbitrum                  13.92 M
    ethereum                  6.19 M
    mode                      1.42 M
    
    total                    21.53 M

File: projects/airpuff/index.js [hunk 1]
```diff
@@ -11,6 +12,19 @@ const contractAbis = {
...
 };
+async function fetchPendlePrice(assetAddress) {
+  // API endpoint template with variable part for the asset address
+  const url = `https://api-v2.pendle.finance/core/v1/1/assets/${assetAddress}`;
+
+  try {
+    const response = await axios.get(url);
+    const accValue = response.data.price.acc;
+    return accValue; // Return the "acc" value for further use
+  } catch (error) {
```
Comment (g1nt0ki) [inline projects/airpuff/index.js:23]: hi, can you skip catching error part? we prefer the adapter to fail, this way we know there is a problem

Comment (g1nt0ki) [reply projects/airpuff/index.js:23]: also, think we price pendle tokens in our coins api

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

File: projects/airpuff/index.js (hunk 1 above)
Comment (0xKMG) [reply projects/airpuff/index.js]: thanks for the review, I've changed the api endpoint to DefiLlama api.

**Comment by llamatester** [comment]
The adapter at projects/airpuff exports TVL: 
        
 
 
    ethereum                  17.66 M
    arbitrum                  11.58 M
    mode                      1.70 M
    
    total                    30.94 M

**Comment by 0xKMG** [comment]
@g1nt0ki let me know if the current implementation is fine, thanks a lot!

**Comment by g1nt0ki** [comment]
@0xKMG I simplified it a bit

**Comment by llamatester** [comment]
The adapter at projects/airpuff exports TVL: 
        
 
 
    ethereum                  17.91 M
    arbitrum                  11.58 M
    mode                      1.76 M
    
    total                    31.25 M

**Pull request merged**